cordova-lib/src/cordova/lazy_load.js

```diff
@@ -147,10 +147,7 @@ function cordova_npm(platform) {
 // Returns a promise that resolves to directory containing the package.
 function npm_cache_add(pkg) {
     var npm_cache_dir = path.join(util.libDirectory, 'npm_cache');
-    // 'cache-min' is the time in seconds npm considers the files fresh and
```

Comment:
How will it work if I remove the caching altogether - sounds like a design change.

It does get a default `cache-min` of 10 seconds which if it is fine for npm, should be fine for us:
https://github.com/npm/npm/blob/2a5977e0c65b244e92d848fcd56f2f80ba8cdf3b/lib/config/defaults.js#L125. Their default scenario is quite similar to ours.